compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/BallerinaSemanticModel.java

```diff
@@ -458,14 +458,17 @@ private BPackageSymbol getModuleSymbol(BLangCompilationUnit compilationUnit) {
     private boolean withinRange(LineRange range, LineRange specifiedRange) {
```

Comment:
Shall we look at getting rid of this method and making use of the `withinRange()` in `PositionUtil`? Don't think this method is properly implemented.

Reply:
@pubudu91, Moved with the new logic. The existing method which was already in the `PositionUtil` was renamed to `isRangeWithinNode()`.